streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignorConfiguration.java

```diff
@@ -153,7 +153,7 @@ public RebalanceProtocol rebalanceProtocol() {
                     throw new IllegalArgumentException("Unknown configuration value for parameter 'upgrade.from': " + upgradeFrom);
             }
         }
-
+        log.info("Cooperative rebalancing enabled now");
```

Comment:
Added for test and thinking this could be useful for users

Reply:
I'm +1 on this: actually I'm thinking it would be nice to make the log message above symmetric, ie something like
`log.info("Eager rebalancing enabled now for upgrade from {}.x", upgradeFrom);`
so we can just grep for "eager" or "cooperative" to figure out what we're on?

Reply:
There's already this log statement https://github.com/apache/kafka/blob/trunk/streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignorConfiguration.java#L150

Did you mean to change the wording of this one?